WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -80,20 +80,25 @@ public WalletViewModel(bool receiveDominant)
 			advancedAction.Items.Add(infoTab);
 			advancedAction.Items.Add(buildTab);
 
-			// Open and select tabs.
+			// Open tabs.
 			sendTab?.DisplayActionTab();
-			if (receiveDominant)
+			receiveTab.DisplayActionTab();
+			coinjoinTab.DisplayActionTab();
+			historyTab.DisplayActionTab();
+
+			// Select tab
+			if (receiveDominant || global.UiConfig.Tab == receiveTab.Title)
 			{
-				coinjoinTab.DisplayActionTab();
-				historyTab.DisplayActionTab();
 				receiveTab.DisplayActionTab(); // So receive should be shown to the user.
 			}
-			else
+			else if (global.UiConfig.Tab == historyTab.Title)
 			{
-				receiveTab.DisplayActionTab();
-				coinjoinTab.DisplayActionTab();
 				historyTab.DisplayActionTab(); // So history should be shown to the user.
 			}
+			else
+			{
+				coinjoinTab.DisplayActionTab(); // So coinjoin should be shown to the user.
```

Comment:
What if it is a HW (watch only)?

Reply:
> What if it is a HW (watch only)?

You mean a HW without the device connected?

The same behavior, it will be open on CoinJoin tab.

Reply:
Umm I think this is probably no important, ok.